From 7d1b7a2cfc5aa6268f8b97eb02dc9aa5a475bb50 Mon Sep 17 00:00:00 2001
From: "Radu Pirea (NXP OSS)" <radu-nicolae.pirea@oss.nxp.com>
Date: Wed, 10 Nov 2021 15:40:50 +0200
Subject: [PATCH 14/19] usb: ehci-mx6: decrease some wait_for_bit_le32 timeouts

Decrease wait_for_bit_le32 timeout to 500ms when it waits for UCMD_RESET
and UCMD_RUN_STOP bits.

Issue: ALB-8074
Upstream-Status: Pending 

Signed-off-by: Radu Pirea (NXP OSS) <radu-nicolae.pirea@oss.nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/usb/host/ehci-mx6.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/host/ehci-mx6.c b/drivers/usb/host/ehci-mx6.c
index 39ac119b7e..e4baac8666 100644
--- a/drivers/usb/host/ehci-mx6.c
+++ b/drivers/usb/host/ehci-mx6.c
@@ -166,12 +166,12 @@ static int usb_phy_enable(int index, struct usb_ehci *ehci)
 
 	/* Stop then Reset */
 	clrbits_le32(usb_cmd, UCMD_RUN_STOP);
-	ret = wait_for_bit_le32(usb_cmd, UCMD_RUN_STOP, false, 10000, false);
+	ret = wait_for_bit_le32(usb_cmd, UCMD_RUN_STOP, false, 500, false);
 	if (ret)
 		return ret;
 
 	setbits_le32(usb_cmd, UCMD_RESET);
-	ret = wait_for_bit_le32(usb_cmd, UCMD_RESET, false, 10000, false);
+	ret = wait_for_bit_le32(usb_cmd, UCMD_RESET, false, 500, false);
 	if (ret)
 		return ret;
 
-- 
2.17.1

